### ISSUE TITLE
tf.Estimator creates loss and loss_1 for eval/train

### ISSUE THREAD

**Issue by blake-varden** [tensorflow/tensorflow#15981]
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
Ubuntu 14.12
- **TensorFlow installed from (source or binary)**:
binary
- **TensorFlow version (use command below)**:
1.4
- **Python version**: 
2.7

### Describe the problem

When using the tf.Estimator, the summary files save out summaries for the loss variable evaluated every checkpoint.  The summary for the training, is saved as 'loss_1' .  I got this tensorboard by running the ciphar10_estimator code located: https://github.com/tensorflow/models/tree/master/tutorials/image/cifar10_estimator/

This makes it difficult to compare the eval/train loss on the same graph in tensorboard.  What causes this naming issue and what can be done to fix it?

Thanks!

![screen shot 2018-01-09 at 12 02 53 pm](https://user-images.githubusercontent.com/22623388/34740703-311a2fc6-f535-11e7-9f88-ab16f65052ee.png)

**Comment by facaiy** [comment]
So you think ˋlossˋ and ˋloss_1ˋ are not good name? What name do you prefer?

**Comment by blake-varden** [comment]
loss is a fine name,  I think they should both be 'loss' so that they appear on the same plot and are able to see how training loss compares to test loss.   I found a fix for the ciphar code by adding a name-scope thats based on the training mode.  the loss_1 name was coming in due to the 'loss' op already existing.

**Comment by facaiy** [comment]
So you request to plot `loss` and `loss_1` in the same figure, right?

**Comment by shivaniag** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
